### PR TITLE
updates

### PR DESCRIPTION
**Pull request by realedk**
Added instructions for Firefox

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,10 +55,10 @@ Just go to https://microsoftedge.microsoft.com/addons/detail/ltt-sidebar-customi
```
Comment (RockSolid1106) [inline README.md:55]: Probably missed the load unpacked line here

File: README.md [hunk 2]
```diff
@@ -28,7 +29,17 @@ Just go to https://microsoftedge.microsoft.com/addons/detail/ltt-sidebar-customi
...
+This extension is also available on the Mozilla Firefox add-ons store!
+Just go to https://addons.mozilla.org/en-US/firefox/addon/customizer-for-ltt-forum/ in Mozilla Firefox and Hit "Add to Firefox".
+![Screenshot (2)](https://user-images.githubusercontent.com/83384667/143074807-3ad7981c-55ab-43b9-a6ee-ed5de82e3127.png)
+
+
+
+
+
+
+# Installation and Setup for Chrome-based browsers
```
Comment (RockSolid1106) [inline README.md:42]: for Chromium-based browsers

File: README.md [hunk 3]
```diff
@@ -44,10 +55,10 @@ Just go to https://microsoftedge.microsoft.com/addons/detail/ltt-sidebar-customi
 
 ![image](https://user-images.githubusercontent.com/84492239/138812632-2890408f-e1d9-432b-938e-37378d7c567b.png)
 
-6. That's it for the installation part. Move over to Customization section. [Customizing the bar](https://github.com/RockSolid1106/LTT-Sidebar-customizer#customizing-the-bar)
+6. That's it for the installation part. Move over to Customization section. [Customizing the bar](https://github.com/RockSolid1106/LTT-Sidebar-customizer/blob/main/README.md#customizing-the-bar)
 
 # Customizing the bar
-1. Navigate to chrome://extensions(or edge://extensions). Now, click "Details" under the extension you just added.
+1. Navigate to chrome://extensions. Now, click "Details" under the extension you just added.
```
Comment (RockSolid1106) [inline README.md:61]: (or edge://extensions)

File: README.md (hunk 2 above)
Comment (RockSolid1106) [inline README.md:42]: Chromium based browsers

**Comment by RockSolid1106** [comment]
Seems your files were old again. I've added the changes separately. Could you check the Chromium branch if the changes consists of all the changes you mentioned? I'll close this if it does.

**Comment by realedk** [comment]
They weren't, I even deleted the repository and re-branched. But I'll close it.

**Pull request closed without merging**